x/ugov/msg_test.go

```diff
@@ -4,77 +4,104 @@ import (
 	fmt "fmt"
 	"testing"
 
+	"github.com/umee-network/umee/v5/util/bpmath"
```

Comment:
This import can be moved to the bottom of the list with the other `umee-network` imports